src/planner/select_plan.go

```diff
@@ -208,6 +208,9 @@ func (p *SelectPlan) Build() error {
 			Range:   segment.Range.String(),
 		}
 		p.Querys = append(p.Querys, tuple)
+		if shardkey == "" {
```

Comment:
It need to optimize: every select cmd to the nonGLOBAL table need to judge，if 32 partitions, 32 times judge are verbose

Reply:
This code has been deleted in the latest commit.